### PR TITLE
[CAL-807] /event-types/single - Fix the header

### LINKED ISSUE #6517: [CAL-807] /event-types/single - Fix the header
Currently

Should be

[View in Figma](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=20%3A26194&t=mVR8DTOGxogtCPnF-1)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-807](https://linear.app/calcom/issue/CAL-807/event-typessingle-fix-the-header)</sub>

### PR DESCRIPTION
**Pull request by gitstart-calcom**
## What does this PR do?

Fixes #6517 

## Demo
https://www.loom.com/share/51c69e059faf41ec92a93eb64de3c720

image

### CODE REVIEW

File: packages/ui/components/button/Button.tsx [hunk 1]
```diff
@@ -210,7 +210,7 @@ export const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, ButtonPr
             <StartIcon
               className={classNames(
                 variant === "icon" && "h-4 w-4",
-                variant === "button" && "h-4 w-4 stroke-[1.5px] ltr:mr-2 rtl:ml-2"
+                variant === "button" && "h-4 w-4 stroke-[1.5px]"
```
Comment (sean-brydon) [inline packages/ui/components/button/Button.tsx:213]: Breaks icon buttons in our app - Please check storybook and [design system](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS?node-id=13743%3A97381&t=6OjJIg9y55PcPjn2-0) when making changes to core components

File: packages/features/shell/Shell.tsx [hunk 2]
```diff
@@ -785,25 +785,24 @@ export function ShellMain(props: LayoutProps) {
...
           <Button
-            variant="icon"
             color="minimal"
             onClick={() =>
               typeof props.backPath === "string" ? router.push(props.backPath as string) : router.back()
             }
             StartIcon={FiArrowLeft}
             aria-label="Go Back"
-            className="ltr:mr-2 rtl:ml-2"
+            className="h-6 w-6 items-center justify-center p-0 ltr:mr-2 rtl:ml-2"
```
Comment (sean-brydon) [inline packages/features/shell/Shell.tsx:797]:
```suggestion
            className="ltr:mr-2 rtl:ml-2"
```

File: packages/features/shell/Shell.tsx [hunk 3]
```diff
@@ -785,25 +785,24 @@ export function ShellMain(props: LayoutProps) {
   const { isLocaleReady } = useLocale();
   return (
     <>
-      <div className="mb-6 flex sm:mt-0 lg:mb-10">
+      <div className="mb-6 flex h-[36px] items-center sm:mt-0 lg:mb-8">
         {!!props.backPath && (
           <Button
-            variant="icon"
             color="minimal"
```
Comment (sean-brydon) [inline packages/features/shell/Shell.tsx:791]: Please update to use our components

variant="icon"
size="sm"

**Review by sean-brydon — CHANGES_REQUESTED** [review]
Thank you for the PR few changes need to be addressed before we can merge

**Comment by PeerRich** [comment]
can you test this with description too? I believe we wanna remove the description right @Jaibles ?

**Comment by gitstart-calcom** [comment]
Currently, with a description
Screenshot 2023-01-30 at 8 11 41 PM

**Comment by PeerRich** [comment]
> Currently, with a description Screenshot 2023-01-30 at 8 11 41 PM

yeah lets hide the description there, because we are adding multi-line and styled descriptions with **bold** and _italic_ text

**Comment by Jaibles** [comment]
@PeerRich correct - there's no description in designs
https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=20%3A26194&t=JxClWoJllqV3zJeC-1

CleanShot 2023-01-30 at 16 50 24@2x

**Comment by gitstart-calcom** [comment]
Hi @PeerRich @Jaibles, the description has been removed now. Please take a look.

**Review by PeerRich — APPROVED** [review]
great stuff!

**Review by alannnc — APPROVED** [review]
Code looks good. Thank you!

**Review by sean-brydon — APPROVED** [review]

**Pull request merged**